src/components/badges/leaderboard-badge.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "../ui/tooltip";
+
+interface LeaderboardBadgeProps {
+  className?: string;
+}
+
+export function LeaderboardBadge({ className = "" }: LeaderboardBadgeProps) {
+  return (
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger>
+          <div
+            className={`inline-flex items-center justify-center w-6 h-6 rounded-full ${className}`}
+          >
+            <img
+              src="/Open_Crosspost-Badge.png"
```

Comment:
Let's have this be the core badge, and remove the three other files.

Have the image source, alt, and tooltip content be determined by the rank